Makefile

```diff
@@ -193,6 +195,8 @@ install: default
 	$(INSTALL) -d $(INST_LUADIR)/apisix/plugins/slslog
 	$(INSTALL) apisix/plugins/slslog/*.lua $(INST_LUADIR)/apisix/plugins/slslog/
 
+	cp -r deps/* /usr/local/
```

Comment:
Why we need this?

All dependencies are installed along side with APISIX when you install it by LuaRocks.

Reply:
All dependencies are installed in INST_LUADIR(/user/share/) by LuaRocks, but when use luajit, it will be find deps in luajit default path `/user/local/...`
This is also why changed INST_LIBDIR and INST_LUADIR in `rockspec/apisix-master-0.rockspec`.